### PR TITLE
Add slideshow

### PR DESCRIPTION
**Pull request by Prashantneu8848**
Adding a slideshow feature to show images of my pet.

### CODE REVIEW

**Comment by Prashantneu8848** [comment]
This PR needs to be updated with previous PR

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,104 @@
...
-</html>
+
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="#">HOME</a>
+      <li class="nav-item">
+        <a class="nav-link" href="#about-me">ABOUT</a>
+      <li class="nav-item">
+        <a class="nav-link" href="#connect">CONNECT</a>
+    </ul>
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html:24]: Nit: add a newline at the end of the file.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,88 @@
...
+  height: 500px;
+}
+
+#about-me,
+#timelapse,
+#pictures,
+#connect {
+  padding: 40px;
+  text-align: center;
+}
```
Comment (corycook) [inline portfolio/src/main/webapp/style.css]: Nit: add a newline at the end of the file.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,104 @@
...
+  </div>
+</div>
+
+<div id="about-me">
+  <h2>About Me </h2>
+  <p>
+    Hi There, I am Prashant Neupane all the way from Nepal. Nepal is a small
+    landlocked country in the Himalayas. I go to Fisk University in Nashville, TN.
+    I am a Computer Science Major and rising Junior.
+  </p>
```
Comment (corycook) [inline portfolio/src/main/webapp/index.html]: Nit: consider removing optional `</p>`. 

See https://html.spec.whatwg.org/multipage/syntax.html#optional-tags and https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

**Review by corycook — APPROVED** [review]
Some minor nits. LGTM

**Pull request merged**